src/app/base/components/AppSideNavigation/AppSideNavigation.tsx

```diff
@@ -138,6 +147,7 @@ const AppSideNavigation = (): JSX.Element => {
                 <AppSideNavItems
                   authUser={authUser}
                   groups={navGroups}
+                  hasVirsh={hasVirsh}
```

Comment:
I think we should avoid adding a prop to the entire group of navigation items where it concerns only a single item.

How about you filter the navigation items array that's passed down to this component instead (making sure it's memoized and not filtered on every render)?